public/app/features/alerting/routes.tsx

```diff
@@ -29,6 +29,20 @@ const alertingRoutes = [
       () => import(/* webpackChunkName: "AlertAmRoutes" */ 'app/features/alerting/unified/AmRoutes')
     ),
   },
+  {
+    path: '/alerting/routes/mute-timing/new',
+    roles: () => ['Admin', 'Editor'],
+    component: SafeDynamicImport(
+      () => import(/* webpackChunkName: "MuteTimings" */ 'app/features/alerting/unified/MuteTimings')
+    ),
+  },
+  {
+    path: '/alerting/routes/mute-timing/:id/edit',
+    oles: () => ['Admin', 'Editor'],
```

Comment:
I think this is a typo? Not sure why the type system didn't pick this up 🤔 

```suggestion
    roles: () => ['Admin', 'Editor'],
```

Reply:
Oops. That's a good catch and I'll add the type